internal/service/main.go

```diff
@@ -25,13 +27,13 @@ type svc struct {
 }
 
 var services = []svc{
```

Comment:
I've mentioned about this approach. It looks compact while calling, but all this `..AsInterface` wraps are awfull.. You may look through gitlab.com/tokend or gitlab.com/distributed_lab groups to find some better approach, inspiration or just come up with personal idea.